controllers/topicPage/TopicPage.js

```diff
@@ -46,7 +47,8 @@ class TopicPage {
       payload.anonymousUserId,
       newTopicPagesWithBlock,
       payload.id,
-      threshold
+      threshold,
+      payload.isBlurredPost
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `getValidActivity` method is now receiving an additional parameter `isBlurredPost`. Ensure that this method is updated to handle this new parameter correctly. If the method doesn't expect this parameter, it could lead to unexpected behavior.



<!-- This is an auto-generated comment by OSS CodeRabbit -->